### ISSUE TITLE
blokada does not autostart

### ISSUE THREAD

**Issue by KennyLeekh** [blokadaorg/blokada#229]
### Steps to reproduce
1. start blokada
2. reboot device
3. blokada does not autostart

### Expected behaviour

blokada will auto start after boot or reboot

### Actual behaviour

blokada does not autostart after boot or reboot

Need to open the app on time to reconnect to vpn

(All setting tried, also tried permanent vpn on vpn setting)

### Configuration
1. Blokada version
3.4.100500 downloaded from F-Droid 

2. Device model
ONEPLUS A6003 (Oneplus 6)

3. Android version / ROM
Oreo 8.1.0 / OxygenOS 5.1.8 (Non-rooted)

### Questions
1. How long have you been using Blokada? 
1 day

2. How long time ago has the problem started to happen? 
Since I started to use

3.  How often does this problem happen to you?
every times I try to reboot the devices

**Comment by rpeter85** [comment]
Do you have other VPN app? Do you know if you have internet connection when your phone boots up? What happens if you disable "Online only" and restart your phone?

**Comment by KennyLeekh** [comment]
Thank you for your reply.

1)Do you have other VPN app?

Originally have dns66, I uninstalled it, reboot and try again, same result

2)Do you know if you have internet connection when your phone boots up?

The WiFi sinigal icon show up in less than 1 second after boot


3)What happens if you disable "Online only" and restart your phone?

Same result, does not autostart until I open the app

**Comment by KennyLeekh** [comment]
Thank you for your advice.

All app ops are on
keep alive is on
always on VPN is enabled

still having the same result, the always on VPN still it was disconnected after reboot.

**Comment by immanuelfodor** [comment]
I can confirm this behavior on Oneplus 5 stock non-rooted OOS. Blokada doesn't autostart, manual start required which is rather annoying. DNS66 autostarts fine.

**Comment by KennyLeekh** [comment]
DNS66 autostart also works fine on my device

**Comment by genius3000** [comment]
FWIW: On OnePlus 5T, autostart works fine for me. App version 3.4.100500 via F-Droid, phone model A5010, OxygenOS 5.1.3, Android 8.1.0.  
No special permissions, battery optimizing enabled, only setting in app enabled is `Start on boot`.

**Comment by piu130** [comment]
Maybe [this](https://stackoverflow.com/questions/7978403/boot-receiver-not-working#9360557) solves the problem.
We should add 
```xml
<action android:name="android.intent.action.QUICKBOOT_POWERON" />
```
to the `core.BootReceiver`.

**Comment by rpeter85** [comment]
Can you all please check if you experience the same behaviour with the new, 3.6 version? The start-on-boot feature is improved there.

**Comment by KennyLeekh** [comment]
This problem seems to be fixed in 3.6 version, auto start works fine even without turning on the "always on VPN" in VPN setting

FYI, I am using Android 9 right now
